doc/api/n-api.md

```diff
@@ -490,6 +490,20 @@ Returns `napi_ok` if the API succeeded.
 
 This API returns true if an exception is pending.
 
+### Fatal Errors
+
+In the event of an unrecoverable error in a native module, a fatal error can be
+thrown to immediately terminate the process.
+
+#### napi_fatal_error
+```C
+NAPI_EXTERN NAPI_NO_RETURN void napi_fatal_error(const char* location, const char* message);
+```
+
+- `[in] location`: The location at which the error occurred.
+- `[in] message`: The message associated with the error.
```

Comment:
Speaking of this, maybe it would be helpful if `message` can be a printf format string instead? That'd invalidate my comment above but would provide callers more flexibility.